docs/documentation/advanced_examples/continuous-diffusive-measurement.md

```diff
@@ -0,0 +1,220 @@
+# Continuous diffusive measurement
+
+!!! Warning "Work in progress."
+    This tutorial is under construction, this is a draft version.
```

Comment:
Do you want to remove this? Or we merge in current state?

Reply:
I'd like to keep it for now, as some aspects of the tutorial are not very rigorous.